CMakeLists.txt

```diff
@@ -494,7 +494,7 @@ if(BUILD_BINDINGS_PYTHON AND NOT BUILD_SHARED_LIBS)
   set(BUILD_BINDINGS_PYTHON OFF)
 endif()
 if(BUILD_BINDINGS_PYTHON)
-  set(BUILD_BINDINGS_PYTHON_VERSION ${PYTHON_VERSION_MAJOR})
```

Comment:
Is this change necessary? `${PYTHON_VERSION_MAJOR}` should be 3.

Reply:
Fixed.